src/Masa.Blazor/Components/Slider/MSlider.cs

```diff
@@ -256,12 +256,9 @@ protected override void OnValueChanged(TValue val)
             //Value may not between min and max
```

Comment:
In line 248. This async method lacks 'await' operators and will run synchronously. Consider using the 'await' operator to await non-blocking API calls, or 'await Task.Run(...)' to do CPU-bound work on a background thread.